girder/api/v1/file.py

```diff
@@ -142,7 +142,7 @@ def requestOffset(self, params):
         upload = self.model('upload').load(params['uploadId'], exc=True)
         offset = self.model('upload').requestOffset(upload)
 
-        if type(offset) is int:
+        if type(offset) in (int, long):
```

Comment:
For even more flexibility, should we be using `isinstance(offset, (int, long))` rather than `type`?  It doesn't matter here, but it would allow subclasses of int and long.  In general, places we use `type` probably should be `isinstance`.